simson/steel/steel_model.py

```diff
@@ -95,25 +95,63 @@ def make_historic_mfa(self) -> InflowDrivenHistoricSteelMFASystem:
         )
 
     def create_future_stock_from_historic(self, historic_in_use_stock):
-        # TODO: predict the In-Use stock according to saturation assumptions similar to Pauliuks 'Steel Scrap Age'. i.e. we need another different extrapolate_stock function
         in_use_stock = extrapolate_stock(
             historic_in_use_stock, dims=self.dims, parameters=self.parameters,
-            curve_strategy=self.cfg.model_customization.curve_strategy,
+            curve_strategy=self.cfg.customization.curve_strategy,
         )
+
         dsm = create_dynamic_stock(
             name='in_use', process=self.processes['use'],
-            ldf_type=self.cfg.model_customization.ldf_type,
+            ldf_type=self.cfg.customization.ldf_type,
             stock=in_use_stock, lifetime_mean=self.parameters['lifetime_mean'],
             lifetime_std=self.parameters['lifetime_std'],
         )
         dsm.compute()  # gives inflows and outflows corresponding to in-use stock
         return FlowDrivenStock(
         stock=dsm.stock, inflow=dsm.inflow, outflow=dsm.outflow, name='in_use', process_name='use',
-        process=self.processes['use'],
-    )
-        """return prepare_stock_for_mfa(
-            dsm=dsm, dims=self.dims, prm=self.parameters, use=self.processes['use']
-        )"""
+        process=self.processes['use'],)
+
+    def extrapolate_trade_to_future(self, future_in_use_stock):
+        """
+        Scale trade flow to the future with new in-use stock inflow and outflow information.
+        TODO: This could be here or within the stock driven MFA. -> Decide
```

Comment:
Maybe it would be nice to have a separate trade module, but I haven't really thought about this and not sure how it would look or interface with the MFA. Logically I think it needs to interface with (or sit inside) the `MFASystem` rather than `SteelModel`, since it could influence and be influenced by stocks and flows in the MFA.compute() method. At the moment (here) you are calculating the trade first and putting it into the params for the MFASystem, right?

Lets discuss future development options, both in terms of methodology and code, together.